### ISSUE TITLE
Contribute more to GF tools like `gftools`, `fontbakery`, `etc`

### ISSUE THREAD

**Issue by eliheuer** [eliheuer/gf-project-plan-q3-q4-2023#12]
I was a much worse programmer and didn't have ChatGPT when I started contributing to `gftools`. I want to take on a more active role contributing to it.

It's also important to reduce the bus-factor on the Google Fonts system and have more people who understands how everything works.

**Comment by davelab6** [comment]
I agree there are opportunities to drive PRs after filing issues that Marc & the wider community agree, this is secondary, but def in scope for 2023
